toshi-moto/docker-compose.yml

```
@@ -0,0 +1,35 @@
+version: "3.7"
+
+services:
+  app_proxy:
+    environment:
+      APP_HOST: toshi-moto_web_1
+      APP_PORT: 8080
```

Use of the right port for connecting to the web interface.
```suggestion
      APP_PORT: 8021
```